### ISSUE TITLE
Error running mypy

### ISSUE THREAD

**Issue by mdp-chris** [matangover/mypy-vscode#87]
I am using VS Code containers. As far as I can tell everything in the container is installed properly.  But I am getting
`Error running mypy in /workspaces: Error: spawn /workspaces/.venv/bin/dmypy ENOENT`
/workspaces/.venv/bin/dmypy exists. The extension is set up in the container (otherwise I wouldn't see an error) mypy is installed using poetry, and as I said, dmypy exists.

So why am I getting this error?

If I run this locally (outside of the container) then everything works properly. I am sure I am missing a setting somewhere.

`"mypy.dmypyExecutable": "${workspaceFolder}/.venv/bin/dmypy",` Is set in the settings file.

**Comment by matangover** [comment]
Thanks for the report. I'll try to reproduce your issue.

**Comment by mdp-chris** [comment]
I figured out what my problem was. So the workspace directory was "/workspace" but I called it "/workspaces" in two places (while it was "/workplace" everywhere else)
I ended up with two directories that looked similar. Both had a dmpy and a python, but the /workspaces on didn't work. Running python from there is what was generating the error. Once I fixed the type, and rebuilt the container, everything worked fine. 
I will close this.
